en/news/_posts/2015-12-25-ruby-2-3-0-released.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: news_post
+title: "Ruby 2.3.0 Released"
+author: "naruse"
+translator:
+date: 2015-12-25 17:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.3.0.
+
+This is the first stable release of Ruby 2.3 series.
+It introduces many new features for example:
+
+A [Frozen String Literal Pragma](https://bugs.ruby-lang.org/issues/11473)
+is introduced.
+With Ruby 2.1, `"str".freeze` has been optimized to reduce object allocation.
+Ruby 2.3 introduces a new magic comment and command line option to
+freeze all string literals in the source files.
+Additionally for debugging, you can get where the object is created on
+`"can't modify frozen String"` error by
+`--debug=frozen-string-literal` command line option.
+
+A [safe navigation operator](https://bugs.ruby-lang.org/issues/11537)
+([so-called lonely operator](https://instagram.com/p/-M9l6mRPLR/)) `&.`,
+which already exists in C#, Groovy, and Swift, is introduced to ease
+`nil` handling as `obj&.foo`. `Array#dig` and `Hash#dig` are also added.
+Note that this behaves as [try! of ActiveSupport](http://api.rubyonrails.org/v4.2.5/classes/Object.html#method-i-try-21),
+which specially handle only nil.
+
+The [did_you_mean gem is bundled](https://bugs.ruby-lang.org/issues/11252).
+The did_you_mean gem
+shows the candidates on the `NameError` and `NoMethodError` to ease
+debugging.
+
+[RubyVM::InstructionSequence#to_binary and .load_from_binary](https://bugs.ruby-lang.org/issues/11788) are introduced as experimental feature. With these features, we can make a ISeq (bytecode) pre-compilation system.
+
+It also includes many performance improvements for example,
+[reconsider method entry data structure](https://bugs.ruby-lang.org/issues/11278),
+[introducing new table data structure](https://bugs.ruby-lang.org/issues/11420),
+[optimize Proc#call](https://bugs.ruby-lang.org/issues/11569),
+machine code level tuning for object allocation and method calling code,
+[smarter instance variable data structure](https://bugs.ruby-lang.org/issues/11170),
+[Socket and I/O allow to use "exception:" keywords for high-performance non-blocking I/O](https://bugs.ruby-lang.org/issues/11229)
+and so on. Check "Implementation improvements" section in NEWS file.
+
+For a complete list of new features and compatibility notes, please see
+[NEWS](https://github.com/ruby/ruby/blob/v2_3_0/NEWS) and
+[ChangeLog](https://github.com/ruby/ruby/blob/v2_3_0/ChangeLog).
+
+With those changes, [2946 files changed, 104057 insertions(+), 59478 deletions(-)](https://github.com/ruby/ruby/compare/v2_2_0...v2_3_0) since Ruby 2.2.0!
+
+Merry Christmas, Happy Holidays, and enjoy programming with Ruby 2.3!
+
+## Download
+
+* <https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.0.tar.bz2>
+  * SIZE:   14185617 bytes
```

Comment:
Need indent and some blank lines like our previous posts :smile: :

<img width="1153" alt="screenshot 2015-12-25 14 45 58" src="https://cloud.githubusercontent.com/assets/1000669/12000701/40a69a1c-ab16-11e5-883a-72954df8fa5f.png">